gravitee-apim-console-webui/src/management/api/api-traffic-v4/analytics/api-analytics-native/api-analytics-native.component.ts

```diff
@@ -70,6 +75,87 @@ export class ApiAnalyticsNativeComponent implements OnInit, OnDestroy {
 
   public activeFilters: Signal<ApiAnalyticsNativeFilters> = computed(() => this.mapQueryParamsToFilters(this.activatedRouteQueryParams()));
 
+  private topRowWidgets: ApiAnalyticsDashboardWidgetConfig[] = [
+    {
+      type: 'stats' as ApiAnalyticsWidgetType,
+      apiId: this.apiId,
+      title: 'Downstream Active Connections',
+      tooltip: 'Number of active downstream connections',
+      analyticsType: 'HISTOGRAM',
+      aggregations: [
+        {
+          type: 'VALUE' as any,
+          field: 'downstream-active-connections' as any,
+        },
+      ],
+    },
+    {
+      type: 'stats' as ApiAnalyticsWidgetType,
+      apiId: this.apiId,
+      title: 'Upstream Active Connections',
+      tooltip: 'Number of active upstream connections',
+      analyticsType: 'HISTOGRAM',
+      aggregations: [
+        {
+          type: 'VALUE' as any,
+          field: 'upstream-active-connections' as any,
+        },
+      ],
+    },
```

Comment:
I will more expect something like this : if doable
```suggestion
    {
      type: 'stats',
      apiId: this.apiId,
      title: 'Upstream Active Connections',
      tooltip: 'Number of active upstream connections',
      analyticsType: 'HISTOGRAM',
      aggregations: [
        {
          type: 'VALUE',
          field: 'upstream-active-connections',
        },
      ],
    } satisfies ApiAnalyticsDashboardWidgetConfig,
```